swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -368,54 +369,69 @@ isolated function removeEmployee(int id) returns int|error {
 
-### Create a service
-To create the service, add the code below to the `service.bal` file. 
+1. Create a `service.bal` file inside the Ballerina package directory (`build_a_data_service`).
 
-```ballerina
-service /employees on new http:Listener(8080) {
-}
-```
+2. Create the service using the [Ballerina HTTP API Designer](/learn/vs-code-extension/design-the-services/http-api-designer/) of the VS Code extension, as shown below.
 
-### Create the resource methods
+    >**Tip:** Use `/employees` as the service path (or the context) of the service, which is attached to the listener listening on port 8080.
 
-Within this service, you can define resource methods to provide access to the database. The code snippet below 
-demonstrates a resource method that can be used to create a new employee via a `POST` request.
+    ![Create the service](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-the-service.gif)
 
-```ballerina
-service /employees on new http:Listener(8080) {
+    The generated REST service will be as follows.
 
-    isolated resource function post .(@http:Payload Employee emp) returns int|error? {
-        return addEmployee(emp);
+    ```ballerina
+    service /employees on new http:Listener(8080) {
+        resource function get .() returns error? {
+        }
     }
+    ```
 
-    isolated resource function get [int id]() returns Employee|error? {
-        return getEmployee(id);
-    }
+### Create the resources
 
-    isolated resource function get .() returns Employee[]|error? {
-        return getAllEmployees();
-    }
+Follow the steps below to define resource functions within this service to provide access to the database.
 
-    isolated resource function put .(@http:Payload Employee emp) returns int|error? {
-        return updateEmployee(emp);
-    }
+1. Create the first resource using the [Ballerina HTTP API Designer](/learn/vs-code-extension/design-the-services/http-api-designer/) of the VS Code extension, as shown below.
 
-    isolated resource function delete [int id]() returns int|error? {
-        return removeEmployee(id);       
+    >**Tip:** Define an HTTP resource that allows the POST operation on the resource path `.` and accepts an `Employee` type payload named `emp`. Use `int|error` as the response type. Complete the generated record by adding the pipe signs to mark the record as a closed one.
+
+    ![Create resource function](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-resource-function.gif)
+
+    The generated resource function will be as follows.
+
+    ```ballerina
+    service /employees on new http:Listener(8080) {
+
+        resource function post .(Employee emp) returns int|error? {
+            return addEmployee(emp);
+        }   
     }
-    
-}
-```
+    ```
+
+2. Similarly, create the other resources as per the code below.
+
+    ```ballerina
+        resource function get [int id]() returns Employee|error? {
```

Comment:
This resource is having a path. parameter. So we better visualize this on designer on how to add path parameter.

Reply:
Done.